tests/topotests/lib/topogen.py

```diff
@@ -849,6 +849,9 @@ def load_frr_config(self, source, daemons=None):
             for daemon in self.RD:
                 # This will not work for all daemons
                 daemonstr = self.RD.get(daemon).rstrip("d")
+                # Special case for bfdd - search for 'bfd' in config files, not 'bf' which is too generic
+                if daemon == self.RD_BFD:
+                    daemonstr = "bfd"
```

Comment:
# Strip the `d` at the end of the name if we have one
                # Special case for bfdd, because `rstrip()` ends up stripping the two ds.
                if daemon == self.RD_BFD:
                    daemonstr = "bfd"
                else
                    daemonstr = self.RD.get(daemon).rstrip("d")

Reply:
done